prov/shm/src/smr_av.c

```diff
@@ -63,7 +63,8 @@ static int smr_av_insert(struct fid_av *av_fid, const void *addr, size_t count,
 	struct smr_ep *smr_ep;
 	struct dlist_entry *av_entry;
 	const char *ep_name;
-	fi_addr_t index, shm_addr = FI_ADDR_UNSPEC;
+	fi_addr_t index;
+	int64_t shm_addr = -1;
```

Comment:
In several places in this patch, int64_t is either 'id' or 'index'.  Here it is still called 'addr'.  Plus the fi_addr_t value is named 'index'.

It may be worth doing another pass through the code to check for naming consistency.